server/routes/v1/deployment.ts

```diff
@@ -151,6 +152,98 @@ export const postDeployment = [
   },
 ]
 
+export const postUngovernedDeployment = [
+  ensureUserRole('user'),
+  bodyParser.json(),
+  async (req: Request, res: Response) => {
+    req.log.info({ code: 'requesting_deployment' }, 'User requesting deployment')
+    const { body } = req
+
+    body.timeStamp = new Date().toISOString()
+
+    const model = await findModelByUuid(req.user, body.modelUuid)
+
+    if (!model) {
+      throw NotFound(
+        { code: 'model_not_found', modelId: body.modelUuid },
+        `Unable to find model with name: '${body.modelUuid}'`
+      )
+    }
+
+    const name = body.name
+      .toLowerCase()
+      .replace(/[^a-z 0-9]/g, '')
+      .replace(/ /g, '-')
+
+    const uuid = `${name}-${nanoid()}`
+
+    const version = await findVersionByName(req.user, model._id, body.initialVersionRequested)
+    if (!version) {
+      throw NotFound(
+        {
+          code: 'version_not_found',
+          modelId: body.modelID,
+          version: body.initialVersionRequested,
+        },
+        `Unable to find version with name: '${body.initialVersionRequested}'`
+      )
+    }
+
+    const versionArray = [version._id]
```

Comment:
This is inaccurate.  There could be multiple versions above the initial version requested.